clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -184,6 +190,20 @@ class TitusClusterProvider implements ClusterProvider<TitusCluster>, ServerGroup
     serverGroup.placement.region = region
     serverGroup.scalingPolicies = serverGroupData.attributes.scalingPolicies
     serverGroup.targetGroups = serverGroupData.attributes.targetGroups
+
+    getTaggedImageAttributes(account, job).ifPresentOrElse(
+      { it ->
+        serverGroup.buildInfo.jenkins =
```

Comment:
I'm assuming that `serverGroup.buildInfo` is never null here right?

Reply:
yes , it either has some details or will be an empty map